src/libs/utils.js

```diff
@@ -45,7 +45,7 @@ const getAllChanges = async ({ rushChangePath, packagePaths, options = {} }) =>
   // Identify changed packages from change logs
   var changedPackages = await getPackagesFromChanges(rushChangePath)
   // Filter projects by version policy
-  if (options.versionPolicy != "") {
+  if (options.versionPolicy && options.versionPolicy != "") {
```

Comment:
Is the second condition required? First one should cover it.

Reply:
This conditioned is required for jest unit testing, because jest will directly use this function and  `options.versionPolicy` will be `undefined`